condor_watch_q.py

```diff
@@ -216,6 +217,26 @@ def parse_args():
     return args
 
 
+def parse_unknown_args(unknown):
+    for unknown_arg in unknown:
+        print("Unknown command.", end=" ")
+        if unknown_arg.isdigit():
+            print("Did you mean condor_watch_q -clusters {}?".format(unknown_arg))
+        elif "-totals" in unknown_arg:
+            print("Did you mean condor_watch_q -no-table?")
+        elif "-userlog" in unknown_arg:
+            print("Did you mean condor_watch_q -files FILE [FILE ...]?")
+        elif "-nobatch" in unknown_arg:
+            print("Defaults to batch, try condor_watch_q -groupby {batch,log,cluster}.")
+        elif "-" in unknown_arg:
+            print(
+                "condor_watch_q does not support the following functionality yet."
+            )
+        else:
+            print("Did you mean -users?")
```

Comment:
Let's do the same trick you did for `-clusters` above and inject the (possible) user names back into the message for copy-pasting.